libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -441,6 +441,11 @@ class AP_AHRS
     // current barometer and GPS altitudes correspond to this altitude
     virtual void set_home(const Location &loc) = 0;
 
+    // set the EKF's origin location in 10e7 degrees.  This should only
+    // be called when the EKF has no absolute position reference (i.e. GPS)
+    // from which to decide the origin on its own
+    virtual bool set_origin(const Location &loc) { return true; }
```

Comment:
I think returning false by default would be better, but we aren't actually doing anything with it.